modules/developer-cli-odo-installing-odo-on-vs-code.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/installing-odo.adoc
+
+[id="installing-odo-on-vs-code"]
+
+= Installing odo on VS Code
+
+The https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector[OpenShift VS Code extension] uses both `odo` and the `oc` binary to interact with your Kubernetes or OpenShift cluster. Install odo on VS Code to work with these features.
+
```

Comment:
This procedure also seems to assume that you already have VS code installed. So I would add that as a prerequisite:

```
.Prerequisites

* You have installed VS Code.
```